Sources/PrimerSDK/Classes/Extensions & Utilities/StringExtension.swift

```diff
@@ -209,6 +209,12 @@ internal extension String {
         let emailP = NSPredicate(format: "SELF MATCHES %@", emailRegEx)
         return emailP.evaluate(with: self)
     }
+    
+    var isValidPhoneNumber: Bool {
+        let phoneNumberRegEx = "^(^\\+628|628)(\\d{8,10})"
```

Comment:
Have you aligned with Android on the regex?

Reply:
Yes - aligned with Android on that, we can keep this one very specific to OVO and then expand to have different validations for other APMs